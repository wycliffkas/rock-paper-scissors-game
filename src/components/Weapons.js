import React from "react";

const Weapons = (props) => {
  return (
    <div>
      <p>Choose weapon</p>
      <button className="weaponBtn" onClick={() => props.selectWeapon("rock")}>
        rock
      </button>
      <button className="weaponBtn" onClick={() => props.selectWeapon("paper")}>
        paper
      </button>
      <button
        className="weaponBtn"
        onClick={() => props.selectWeapon("scissors")}
      >
        scissor
      </button>
    </div>
  );
};

export default Weapons;
