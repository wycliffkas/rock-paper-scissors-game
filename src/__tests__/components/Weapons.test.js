import React from "react";
import { shallow } from "enzyme";
import Weapons from "../../components/Weapons";

describe("Weapons", () => {
  const wrapper = shallow(<Weapons />);
  it("should render without crashing", () => {
    expect(wrapper).toMatchSnapshot();
  });
});
