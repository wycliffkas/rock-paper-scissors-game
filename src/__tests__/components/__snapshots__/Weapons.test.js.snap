// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Weapons should render without crashing 1`] = `ShallowWrapper {}`;
